docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -20,10 +20,11 @@ The minimal required version of .NET Framework is 4.7.2.
     - Ensure Visual Studio is on Version "16.8" or greater
     - Ensure "Use previews of the .NET Core SDK" is checked in Tools -> Options -> Environment -> Preview Features
     - Restart Visual Studio
-1. [.NET 6.0 Preview 1 SDK](https://dotnet.microsoft.com/download/dotnet-core/6.0) [Windows x64 installer](https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-6.0.100-preview.1-windows-x64-installer)
```

Comment:
Why this line is being removed? I'd rather download/install the sdk once for all of git worktrees. Restore doesn't do that.

Reply:
@alrz A few lines later, I linked to the SDK download page link stating that it's not a requirement if using Visual Studio with the new script since Restore.cmd would have already downloaded it to `.dotnet`. Any other work without startvs.cmd will certainly need the SDK to be installed the normal way.